src/DynamicMvvm/Property/DynamicProperty.cs

```diff
@@ -17,7 +19,8 @@ public class DynamicProperty : IDynamicProperty
 		/// </summary>
 		/// <param name="name">Name</param>
 		/// <param name="value">Initial value</param>
-		public DynamicProperty(string name, object value = default)
```

Comment:
If you change the method signature, it's a breaking change.
You should instead add new overloads just to avoid the breaking changes.

Reply:
We should probably update docs, if needed. Didn't think of that.

Reply:
I figured that since I've added it as an optional parameter that it didn't count as breaking changes

Reply:
That's also true I guess.

Reply:
(This is true for all cases where you're adding a parameter to a method, even if the parameter has a default value.)

Reply:
Fair enough. I did just change the description of the PR. Although, just to be sure, should we go with the different constructor here or optional parameter are fine here?

Also for the doc part @Soap-141, I've made sure to add xml doc for the parameter I've added. Although, I should do it for the new constructor on the factory, I'm not sure if we need to add explicit documentation for that

Reply:
Add new overloads instead of the optional parameter.

We can do the optional parameter later, in a major release with other breaking changes. We could start a new branch for that. (We'll probably need one anyways for Uno 5.)